source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,35 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+  template <class Policy, class InputIt, class OutputIt, class UnaryOp, class UnaryPredicate>
+  OutputIt
```

Comment:
For a few other APIs here, the returned value is described - perhaps makes sense to do for `transform_if` as well.

Reply:
I've added a sentence about the return for each overload, thanks.